README.md

```diff
@@ -213,6 +213,23 @@ export GITLAB_PROJECT_ID=<GITLAB_PROJECT_ID>
 export SOURCE_CODE_PROVIDER_URL=<https://git.yourdomain.com>
 ```
 ​
+### Reporting configuration
+​
+* Optional. Defines execution environment for report statistics:
+```bash
+export EXECUTION_ENV=buddyCI
+```
+​
+* Optional. Enable the report sending functionality. it’s disabled by default.
+```bash
+export REPORTING_ENABLED=1
```

Comment:
Documentation for `INTEGRATOR_ENABLED` says that it can be activated with `true`:

```
export INTEGRATOR_ENABLED=true
```

Here we activate it with 1. It will be good if we will have consistency in the type of activation values.

Reply:
- [ ] Please use true for example here